test/hotspot/jtreg/runtime/os/TestTrimNative.java

```diff
@@ -296,12 +295,16 @@ public static void main(String[] args) throws Exception {
 
             case "trimNativeLowInterval":
             case "trimNativeLowIntervalStrict": {
+                long ms1 = System.currentTimeMillis();
                 OutputAnalyzer output = runTestWithOptions(
                         new String[] { "-XX:+UnlockExperimentalVMOptions", "-XX:TrimNativeHeapInterval=1" },
                         new String[] { TestTrimNative.Tester.class.getName(), "0" }
                 );
+                long ms2 = System.currentTimeMillis();
+                long runtime_ms = ms2 - ms1;
+                long maxTrimsExpected = runtime_ms;
```

Comment:
Fixes the style and improves code a bit:

```suggestion
                int maxTrimsExpected = (int)(ms2 - ms1);
```

Then `(int)maxTrimsExpected` does not need a cast too.